Gemfile.lock

```diff
@@ -276,10 +313,15 @@ DEPENDENCIES
   rspec-rails
   sass-rails (~> 5.0)
   sdoc (~> 0.4.0)
+  sidekiq
   slim-rails
   spring
   state_machines-activerecord
   therubyracer
   turbolinks
   uglifier (>= 1.3.0)
+  vacuum
   web-console (~> 2.0)
+
+BUNDLED WITH
+   1.11.2
```

Comment:
bundler :new_moon_with_face: 

imo not worth it to spend time trying to fix this between systems, even if it's a bit of a nuisance that it keeps showing up, it doesn't really hurt anything to keep checking in / removing / etc